telegram_export/downloader.py

```diff
@@ -80,7 +80,8 @@ def _check_media(self, media):
         """
         Checks whether the given MessageMedia should be downloaded or not.
         """
-        if not media or not self.max_size:
+        # It is needed to chek the size with the max_size defined in config file
+        if not media or media.document.size > self.max_size:
```

Comment:
Did you check at least twice the Telethon documentation to make sure `document is not None` and `size is not None` (or they can't be `None`, not present at all)? We don't want an error here.

Reply:
I will fix it, let you know the result

Reply:
There are some point need consideration, Media has multiple type and it is not easy to get the size of each object, I mean it is depended to the Type and calling media.document.size will only works for document type and not other. I think we need to use polymorphism to do that. each type has to implement an interface for general attributes like size, so calling somthing like media.getsize() return the size of file regardless the type of it,

Reply:
> each type has to implement an interface for general attributes like size

More like some utils method `get_size(media)`, unless you're willing to do this change in Telethon which would be by far harder :)

Reply:
yes, I found an easier way to do that, but not sure yet, so far I realized that just document types have size attribute, so when calling export_utils.get_media_type(media) returns "document" we can refer to media.document.size and compare it to the max_size,
What's your idea?

Reply:
also in utils there is an method named get_file_location(media):, So already it has been implemented, I will use it and  after test I will commit the code

Reply:
Just add another function to [`utils.py`](https://github.com/expectocode/telegram-export/blob/master/telegram_export/utils.py) to `get_file_size` for media, comparing media type with `isinstance`.

Reply:
its done !, how can I pull the changes ?